pkg/process/checks/container_rt.go

```diff
@@ -40,6 +41,12 @@ func (r *RTContainerCheck) RealTime() bool { return true }
 // Run runs the real-time container check getting container-level stats from the Cgroups and Docker APIs.
 func (r *RTContainerCheck) Run(cfg *config.AgentConfig, groupID int32) ([]model.MessageBody, error) {
 	ctrList, err := util.GetContainers()
+
+	if err == containercollectors.ErrPermaFail || err == containercollectors.ErrNothingYet {
+		log.Debug("container collector was not detected, container check will not return any data")
```

Comment:
Make this `log.Trace` or else we might see a log entry every 2s (on RT) and 10s on regular

Reply:
Good point, done